src/Spice86.Core/Emulator/Devices/Video/Registers/IoAddressSelect.cs

```diff
@@ -0,0 +1,6 @@
+namespace Spice86.Core.Emulator.Devices.Video.Registers;
+
+public enum IoAddressSelect {
```

Comment:
I dunno. I'd probably move this inside the `Spice86.Core.Emulator.Devices.Video.Registers.General.MiscellaneousOutput` class as `IoAddressSelectValue`, since it has no practical use outside the MiscellaneousOutput register scope.

same for Polarity and ClockSelect

Reply:
This shouldn't have moved compared to master ?

Reply:
Done. Enums moved to the MiscOutput class.